openml/extensions/sklearn/extension.py

```diff
@@ -476,6 +477,35 @@ def _is_sklearn_flow(cls, flow: OpenMLFlow) -> bool:
             or ',sklearn==' in flow.external_version
         )
 
+    def _get_sklearn_description(self, model: Any, char_lim: int = 1024) -> str:
+        '''Fetches the sklearn function docstring for the flow description
+
+        Parameters
+        ----------
+        model: The sklearn model object
+        char_lim: int, specifying the max length of the returned string
+            OpenML servers have a constraint of 1024 characters for the 'description' field.
+
+        Returns
+        -------
+        string of length <= char_lim
+        '''
+        def match_format(s):
+            return "{}\n{}\n".format(s, len(s) * '-')
+        s = inspect.getdoc(model)
+        if len(s) <= char_lim:
+            return s
+        try:
+            pattern = "Read more in the :ref:"  # "Parameters"
+            index = s.index(pattern)
+        except ValueError:
+            pattern = "Parameters"
```

Comment:
What would happen if that doesn't exist either, for example due to a mistake in the sklearn docs? Maybe this should be more failure tolerant and use the whole string if the two patterns aren't part of the docstring.

Reply:
Are nested try...except an accepted practice?

Reply:
"Flat is better than nested" from [The Zen of Python](https://www.python.org/dev/peps/pep-0020/#id3) :)
Followed this...